### ISSUE TITLE
Create Roadmap View

### ISSUE THREAD

**Issue by vincent6767** [o3de/sig-testing#61]
Hi! SIG release is working to help the O3DE community to get visibility of the O3DE roadmap. Full context can be read here: https://github.com/o3de/sig-release/issues/79. 

In order to achieve that goal, we need your help to create the roadmap for your SIG by **February 6th, 2023**, and start giving brief updates about the roadmap items at the **Joint TSC meeting on February 28th, 2023**. Instruction to create the roadmap view can be read in the RFC section "Roadmap Review in TSC monthly meeting".

Let me know if you have any concerns with the dates or any questions about the ask!

**Comment by Kadino** [comment]
Board has been created at https://github.com/orgs/o3de/projects/35/views/1 however is currently set to private. I am unable to modify the visibility setting to public:
```Visibility
Only organization owners can change project visibility.```

Will search for an admin in the upcoming week.

**Comment by Kadino** [comment]
Notably there appears to be no built-in ability to automatically add all issues as they get tagged with the `kind/roadmap` label.

**Comment by vincent6767** [comment]
> Notably there appears to be no built-in ability to automatically add all issues as they get tagged with the kind/roadmap label.

Yes. This has been raised by other SIGs. I'll add this to the next improvement idea. 
https://github.com/o3de/sig-release/issues/79#issuecomment-1381371262

**Comment by Kadino** [comment]
In the meantime, roadmap can be summarized by issue query: https://github.com/o3de/sig-testing/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Froadmap

**Comment by vincent6767** [comment]
This is a neat trick. Thanks for sharing @Kadino !

**Comment by Kadino** [comment]
Roadmap is public now, thanks @amzn-changml !
